VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+`9*R + R + R*5% = 1005 ⇔ R = 1005/10.05 = 100`
+
+* For the proposer validator
+  * The pool obtains R + R*5% = 105 ATOMs
+  * Commission = 105*80%*1% = 0.84 ATOMs
+  * Validator reward = 100 * 20% + Commission = 21.84 ATOMs
+  * Delegators rewards = 105 * 80% - Commission = 83.16 ATOMs (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+* For each non-proposer validator
+  * The pool obtains R = 100 ATOMs
+  * Commission = 100*80%*1% = 0.8 ATOMs
+  * Validator reward = 100 * 20% + Commission = 20.8 ATOMs
+  * Delegators rewards = 100 * 80% - Commission = 79.2 ATOMs (each delegator will be able to claim its portion of these rewards in proportion of their stake)
+
+### What are the slashing conditions?
+
+If a validator misbehaves, its bonded stake along with its delegator’s stake and will be slashed. The severity of the punishment depends on the type of fault. There are 3 main faults that can result in loss of funds for a validator and its delegators :
+
+* **Double signing:** If someone reports on chain A that a validator signed two blocks at the same height on chain A and chain B, this validator will get slashed on chain A
+* **Unavailability:** If a validator’s signature has not been included in the last X block, the validator will get slashed by a marginal amount proportional to X. If X is above a certain limit Y, then the validator will get unbonded
+* **Non-voting:** If a validator did not vote on a proposal and once the fault is reported by a someone, his stake will be a little slashed. He will also get a penalty. Once a validator gets a penalty, no additional penalty can be submitted for a given period (3 weeks). After this period, if a second penalty is submitted, the validator gets a second light slash and gets unbonded. If, after the first penalty, no other penalty is submitted for a 3 months period, then the penalty can be cleared by sending a special transaction.
+
+Note that even if a validator does not intentionally misbehaves, it can still be slashed if its node crashes, looses connectivity, gets DDOSed or if its private keys are compromised.
+
+### Incentives for validator #101 to stake?
+
+Validator candidates that do not have enough total stake to be in the validator set will have to run validator operation (or at least be ready to) without getting any revenue. 
+
+There are solutions that are considered to mitigate this issue. For example, we could create a sub-layer of delegation. Validator candidate could delegate to existing validators while they’re trying to attract delegators, thereby generating revenue. At the same time, delegators already bonded to validators could signal their intention to switch to this validator candidate should it get enough stake to enter the validator set. When the validator candidate would get enough pledge, it would transition to a validator and all the delegator who pledged would join its staking pool.
+
+### Do validators need to self-bond ATOMs?
```

Comment:
add the point that  "no they don't"

Reply:
Talk about how reputation can be put on the line alternatively right.

Reply:
addressed